src/test/ts/browser/EditorBehaviorTest.ts

```diff
@@ -114,8 +114,8 @@ describe('EditorBehaviourTest', () => {
         });
         eventStore.clearState();
         ctx.editor.setContent('<p>Initial Content</p>');
-
         await ctx.reRender({ onSetContent: eventStore.createHandler('NewHandler') });
+        await Waiter.pWait(0);
```

Comment:
Could we add a comment in the code as to why this is needed - just for future understanding